src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj

```diff
@@ -330,12 +433,54 @@
 			productReference = 3B0347481F212F0100C8EF1F /* VpnExtension.appex */;
 			productType = "com.apple.product-type.app-extension";
 		};
+		A26D262C2A1C41B1009838E0 /* OutlineLauncher */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = A26D26392A1C41B4009838E0 /* Build configuration list for PBXNativeTarget "OutlineLauncher" */;
+			buildPhases = (
+				A26D26292A1C41B1009838E0 /* Sources */,
+				A26D262A2A1C41B1009838E0 /* Frameworks */,
+				A26D262B2A1C41B1009838E0 /* Resources */,
+				A2B8C1502A58B70E0054487E /* Embed PlugIns */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				A2B8C14F2A58B70E0054487E /* PBXTargetDependency */,
+			);
+			name = OutlineLauncher;
+			packageProductDependencies = (
+				A271D41F2A7069D8009981B2 /* OutlineLauncher */,
+			);
+			productName = OutlineLauncher;
+			productReference = A26D262D2A1C41B1009838E0 /* OutlineLauncher.app */;
+			productType = "com.apple.product-type.application";
+		};
+		A2DBB2F52A00D2DD0017E696 /* AppKitBridge */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = A2DBB2F72A00D2DD0017E696 /* Build configuration list for PBXNativeTarget "AppKitBridge" */;
+			buildPhases = (
+				A2DBB2F22A00D2DD0017E696 /* Sources */,
```

Comment:
Do you need to specify sources for the `AppKitBridge`? Aren't all sources in the `OutlineAppleLib`?

Reply:
That's odd. For the VpnExtension we just link the library. I'm not sure why this would be different for the AppKitBridge.